src/mainWindow.js

```diff
@@ -41,12 +41,12 @@ setInterval(() => {
 
   host.insertAdjacentElement('afterend', oaVersion);
 
-  let advanced = document.querySelector('[class*="socialLinks"]').parentElement.querySelector('[class*="premiumTab"] ~ [class*="header"] + [class*="item"] + [class*="item"] + [class*="item"] + [class*="item"] + [class*="item"] + [class*="item"] + [class*="item"] + [class*="item"] + [class*="item"] ~ [class*="separator"]')?.previousSibling;
-  if (!advanced) advanced = [...document.querySelectorAll('[class*="item"]')].find(x => x.textContent === 'Advanced');
```

Comment:
Keep this fallback?

Reply:
i think a fallback that's language exclusive doesn't make a lot of sense as there is already a fallback which is clicking on the version, but sure if you want, i could keep it

Reply:
i think a fallback that's language exclusive doesn't make a lot of sense as there is already a fallback which is clicking on the version, but sure if you want, i could keep it